api/migrations/20210816203116-create-bean-log.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("BeanLogs", {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.literal("uuid_generate_v4()"),
+      },
+      beanTypeId: {
+        type: Sequelize.UUID,
+        references: {
+          model: "beanTypes",
+          key: "id",
+        },
+      },
+      userId: {
+        type: Sequelize.STRING,
+        references: {
+          model: "users",
+          key: "id",
+        },
+      },
+      givenBy: {
+        type: Sequelize.STRING,
```

Comment:
Why don't we reference user ID here? Is it because some discord users might not have signed up for a user profile?